stardew-access/Patches/MenuWithInventoryPatches/ItemGrabMenuPatch.cs

```diff
@@ -106,11 +106,11 @@ private static bool NarrateHoveredButton(ItemGrabMenu __instance, int x, int y)
                         toSpeak = $"{toSpeak} Selected";
                     goto SayWithChecker;
                 }
+                return false;
             }
             else
```

Comment:
I think we can move `__instance.discreteColorPickerCC != null` to the for loop itself and make the previous one only else, lemme check if that works.

Reply:
Yup, making the last else if as else and having this condition in for loop fixes it, `__instance.discreteColorPickerCC != null && i < __instance.discreteColorPickerCC.Count`, will commit the changes.